### PR TITLE
[REM - 2009] Mohammed imran

### PR DESCRIPTION
**Pull request by 14imran**
1 iteration is completed but second iteration not done. I was not able to break it down.

### CODE REVIEW

File: javascript/chronometer.js [hunk 1]
```diff
@@ -1,26 +1,63 @@
...
     // ... your code goes here
+    setInterval(() => {
+      callback();
+      this.currentTime += 1;
+    },1000);
   }
   getMinutes() {
     // ... your code goes here
+
+let minutes = 0;
```
Comment (IH-WebDev-TA-Remote) [inline javascript/chronometer.js:21]: check indentation of elements here, they should look inside the function for readability.

File: javascript/chronometer.js [hunk 2]
```diff
@@ -1,26 +1,63 @@
...
+    // let sec = Math.abs(this.currentTime)
+    return sec;
   }
-  twoDigitsNumber() {
+  twoDigitsNumber(input) {
     // ... your code goes here
+    if(input<10)
+    {
+      return "0" + input;
+    }
```
Comment (IH-WebDev-TA-Remote) [inline javascript/chronometer.js:40]: check the correct formatting of if conditionals, also some spacing for the condition inside like this:

```
if (input < 10) {
      return "0" + input;
    }
```

File: javascript/chronometer.js (hunk 1 above)
Comment (14imran) [reply javascript/chronometer.js:21]: okay. i will check it.

**Comment by IH-WebDev-TA-Remote** [comment]
Good solution for iteration 1 and I also see you managed to link them and run the chronometer, but still a lot of code there missing for iteration 2. Go back to it even with the solution to see how the DOM manipulation part and use of methods is done. Keep it up!

**Pull request closed without merging**